docs/CONTRIBUTING.md

```diff
@@ -1,90 +1,158 @@
-# Contribution Guidelines
+# Gitbase Playground Contribution Guidelines
 
 As all source{d} projects, this project follows the
 [source{d} Contributing Guidelines](https://github.com/src-d/guide/blob/master/engineering/documents/CONTRIBUTING.md).
 
+In addition to those guidelines, this document describes how to build and run the application for development purposes.
 
-## Additional Contribution Guidelines
+# Requirements
 
-In addition to the [source{d} Contributing Guidelines](https://github.com/src-d/guide/blob/master/engineering/documents/CONTRIBUTING.md),
-this project follows the guidelines described below.
+Refer to the main [README](../README.md) for the common requirements and configuration environment variables.
 
+Make sure you have [installed Go](https://golang.org/doc/install) and properly [configured the $GOPATH](https://github.com/golang/go/wiki/SettingGOPATH). The project should be cloned in `$GOPATH/github.com/src-d/gitbase-playground`.
 
-# Architecture
+You also need to install [Node.js](https://nodejs.org) and [Yarn](https://yarnpkg.com/en/docs/install).
```

Comment:
Do you think this section would be easier to grasp at the first glance, if structured as a Markdown list ?

Reply:
:+1: It looks better now as a list in 3a37a77